### ISSUE TITLE
neo4j-admin import failed when header name contains colon(url)

### ISSUE THREAD

**Issue by sjnov11** [neo4j/neo4j#13014]
## Guidelines

- Neo4j version: 5.3.0
- Operating system: CentOS 7
- API/Driver: (for example Cypher/Java API/Python driver vX.Y.Z)
- **Steps to reproduce**

Hello,
I am trying to upgrade my neo4j version to Neo4j 5.3.0:latest.
When i am trying to import csv data using neo4j-admin import, it fails as below, which was not occured on 4.4.11.

```
org.neo4j.internal.batchimport.input.HeaderException: 
Unable to parse header, unknown property type 
'://example.com/property/name:string[]
```
and this is how my node header file looks like.
```
uri:ID    http://example.com/property/name:string[]
```

It seems property name (that is URL) cause an error, because it contains 'colon(:)'.
as header contains information for each field, with the format <name>:<field_type>

Is there any way to import data successfully?

**Comment by tinwelint** [comment]
Hi @sjnov11 thanks for reporting. I can see the problem and this is an oversight when adding functionality to the CSV header. From what I can tell this can be solved by looking at the last `:` and use that as the type definition, I tried locally and it worked. This will hopefully reach 5.4 or 5.5.

**Comment by tinwelint** [comment]
This fix is in 5.5 and above
